lib/encoder.py

```diff
@@ -549,6 +549,10 @@ def AllScoredVariants(self, videofile, vary):
                                                          videofile).encodings)
     return encodings
 
+  def ChangeValue(self, name, new_value):
```

Comment:
ChangeValue implies you mutate the current encoder. What about CreateVariant(self, parameter_to_change, new_value)?